src/vs/base/browser/ui/splitview/paneview.css

```diff
@@ -21,7 +21,7 @@
 }
 
 .monaco-pane-view .pane > .pane-header {
-	height: 22px;
+	height: 23px;
```

Comment:
We should not go away from our `22px` assumption in titles and headers. This looks like a hack to me.